hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RPC.java

```diff
@@ -896,15 +896,26 @@ public Builder setNumHandlers(int numHandlers) {
       this.numHandlers = numHandlers;
       return this;
     }
-    
-    /**
+
+    /**@deprecated call {@link #setNumReaders(int value)} instead.
```

Comment:
can you put that on the line below.

Reply:
Thanks @steveloughran help review, i will update PR later.